service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/service/EncounterReportBundleService.java

```diff
@@ -34,15 +37,15 @@ public Bundle createEncounterBundle(POCDMT000002UK01ClinicalDocument1 clinicalDo
         addParticipants(bundle, encounter);
         addAppointment(bundle, encounter);
         addLocation(bundle, encounter);
-        addPatient(bundle, encounter);
+        addSubject(bundle, encounter);
         addEpisodeOfCare(bundle, encounter);
 
         return bundle;
     }
 
-    private void addPatient(Bundle bundle, Encounter encounter) {
-        addEntry(bundle, encounter.getSubjectTarget());
-    }
+//    private void addPatient(Bundle bundle, Encounter encounter) {
```

Comment:
Why is this commented out?

Reply:
This method is no longer required - it was a temporary fix for addSubject()